src/Compilers/CSharp/Portable/Symbols/Source/FieldSymbolWithAttributesAndModifiers.cs

```diff
@@ -88,7 +90,9 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 Debug.Assert(completed);
             }
 
+#nullable disable // Can '_lazyCustomAttributesBag' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
I don't think so. Looking at `LoadAndValidateAttributes`, it always sets its `ref CustomAttributesBag<CSharpAttributeData> lazyCustomAttributesBag` parameter, except when `earlyDecodingOnly` is `true` (which is not the case here).